### ISSUE TITLE
DMs don't show inline images

### ISSUE THREAD

**Issue by LakesGeek** [baedert/corebird#626]
Where the main tweet tab shows any images inline, the DM tab does not.  As someone with a friend who sends a lot of images (and sometimes videos) this would be useful to have.

(Aside, great work on the client, it's really come a long way since I last commented on it)

**Comment by Vistaus** [comment]
You're right that it doesn't. But it's probably a client API limit as they aren't shown in the official Twitter client on my phone either - I have to open up the browser in desktop mode to view them.

**Comment by baedert** [comment]
Media are indeed attached to the DM json, but it's hard to cache them along with the actual message text. On the other hand we should already be able to simply load media if it's contained in a pic.twitter.com url and the text contains that url, but we also can't really cache the url entities for that.
